jcl/src/java.base/share/classes/jdk/internal/ref/CleanerShutdown.java

```diff
@@ -47,7 +47,14 @@ public static void shutdownCleaner() {
 			}
 		}
 
-		/*[IF JAVA_SPEC_VERSION < 24]*/
+		/*[IF JAVA_SPEC_VERSION >= 24]*/
+		try {
+			Method cleanableListReset = CleanerImpl.CleanableList.class.getDeclaredMethod("reset"); //$NON-NLS-1$
+			cleanableListReset.setAccessible(true);
+			while (!commonCleanerImpl.activeList.isEmpty()) {
```

Comment:
Why a while loop? reset() sets the list to empty.

Reply:
Removed the `while` loop.